qulice-ant/src/main/java/com/qulice/ant/QuliceTask.java

```diff
@@ -119,15 +124,9 @@ public void execute() {
      * @return Environment.
      */
     private Environment environment() {
-        if (this.sources == null) {
-            throw new BuildException("srcdir not defined for QuliceTask");
-        }
-        if (this.classes == null) {
-            throw new BuildException("classesdir not defined for QuliceTask");
-        }
-        if (this.classpath == null) {
-            throw new BuildException("classpath not defined for QuliceTask");
-        }
+        this.assertNotNull(this.sources, "sources");
```

Comment:
@simonjenga What is this for? Submit a separate pull request for this, don't include it here.